src/custom/components/TransactionSettings/TransactionSettingsMod.tsx

```diff
@@ -245,7 +245,11 @@ export default function TransactionSettings({ placeholderSlippage }: Transaction
             <QuestionHelper
               bgColor={theme.bg3}
               color={theme.text1}
-              text={t`Your swap expires and will not execute if it is pending for longer than the selected duration. ${INPUT_OUTPUT_EXPLANATION}`}
+              text={
+                <Trans>
+                  Your swap expires and will not execute if it is pending for longer than the selected duration.
```

Comment:
You can keep the explanation, it just wouldn't work with `${INPUT_OUTPUT_EXPLANATION}`, but it will with `{INPUT_OUTPUT_EXPLANATION}`